.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.18.0
+current_version = 0.19.0
```

Comment:
We generally don't bump versions until release (our OS libraries also tend to follow (git-flow)[https://nvie.com/posts/a-successful-git-branching-model/], making them a bit of an oddball.

Regardless, the flow here should be simply making the change, merging to develop, then releasing soon after.

Reply:
I can revert this change. If we don't bump the version, can Styx still get this newer version?

Reply:
To release your changes, you'll first merge your changes back to develop, after which we'll go through the normal release flow.